packages/react-components/src/components/MessageThread.tsx

```diff
@@ -1067,7 +1065,6 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
             message={messageProps.message}
             userId={props.userId}
             remoteParticipantsCount={participantCount ? participantCount - 1 : 0}
-            inlineAcceptRejectEditButtons={!isNarrow}
```

Comment:
The `isNarrow` rendering is something that needs to be put back on from the initial PR which didn't implement this

Reply:
no, in this case we don't need it. It will be needed for the ChatMessage, that's why I haven't removed it fully.